packages/editor/src/client/panel/ExportSubscriptionsCsvModal.tsx

```diff
@@ -24,24 +43,43 @@ export function SubscriptionAsCsvModal() {
     <>
       <div style={{display: 'flex', justifyContent: 'flex-end'}}>
         <Button appearance="primary" onClick={() => getSubsCsv()}>
-          {t('userList.panels.exportSubscriptions')}
+          {t('userList.panels.generateCSV')}
         </Button>
       </div>
+      <Divider>{t('userList.panels.actions')}</Divider>
+      <div style={{display: 'flex', justifyContent: 'space-around'}}>
+        <IconButton
+          appearance="primary"
+          icon={<Icon size="lg" icon="download" />}
+          disabled={!subscriptionsCsvData?.subscriptionsAsCsv}
+          onClick={() =>
+            downloadBlob({
+              content: subscriptionsCsvData?.subscriptionsAsCsv || '',
+              filename: 'export.csv',
+              contentType: 'text/csv;charset=utf-8;'
+            })
+          }>
+          {t('userList.panels.download')}
+        </IconButton>
+        <IconButton
+          appearance="primary"
+          icon={<Icon size="lg" icon="copy" />}
+          disabled={!subscriptionsCsvData?.subscriptionsAsCsv}
+          onClick={() => navigator.clipboard.writeText(subscriptionsCsvData!.subscriptionsAsCsv!)}>
```

Comment:
Is the `!` necessary here? Can you get around it with `|| ' '` instead?

Reply:
yeah these kind of assertions are all through the project and my thought was that we should get rid of them all together